README.md

```diff
@@ -1,6 +1,5 @@
 # GitLabChangelog
 [![CI](https://github.com/invenia/gitlabchangelog/workflows/CI/badge.svg)](https://github.com/invenia/gitlabchangelog/actions?query=workflow%3ACI)
-[![Coverage Status](https://coveralls.io/repos/github/invenia/gitlabchangelog/badge.svg?branch=master)](https://coveralls.io/github/invenia/gitlabchangelog)
```

Comment:
I guess we're not actually publishing the coverage data and thats why this is removed?

Reply:
Yes, I tried briefly looking into it but was confused and don't know how to publish it to coveralls

Reply:
I think someone added this in, then didn't add it to Coveralls. Or someone removed it at some point in time as it DNE,

> https://coveralls.io/github/Invenia/
> https://coveralls.io/github/invenia/gitlabchangelog

Reply:
It's added by cookie cutter but doesn't get set up by cookie cutter